function.dd

```diff
@@ -2076,6 +2160,83 @@ const int x = foo("1");
         generated. A function template would be the appropriate
         method to implement this sort of thing.)
 
+$(H3 $(LNAME2 nogc-functions, No-GC Functions))
+
+    $(P No-GC functions are functions marked with the $(D @nogc) attribute.
+        That function does not allocate memory on the GC heap,
+        through following language features:
+    )
+
+    $(UL
+    $(LI $(DDSUBLINK expression, ArrayLiteral, constructing array) on heap)
+    $(LI resizing array $(D .length))
+    $(LI array $(DDSUBLINK expression, CatExpression, concatenation) and appending)
+    $(LI $(DDSUBLINK expression, AssocArrayLiteral, constructing associative array) on heap)
+    $(LI $(DDSUBLINK expression, IndexExpression, indexing) associative array
+        so it may throw $(I RangeError) if the specified key is not associated with)
```

Comment:
`indexing) an associative array because it may throw $(I RangeError) if the specified key is not present`